### DISCUSSION TITLE
Overlay expander over other cards

### DISCUSSION THREAD

**Discussion by Mattat01** [MelleD/lovelace-expander-card#374] [Q&A] [answered]
Hi,

Love your work, thanks so much!

I was wondering if there is any way to make the expander overlay the other cards when expanded, instead of pushing everything down to make space? If not, I'm looking at moving it to a sidebar, but I'd really like to stay with a sections view. I'm using the expander for an 'add event' button for a calendar and want the input form to expand over the calendar instad of pushing it lower on the page. I've not seen any eveidence that anything other than a popup can even do this, but I thought I'd ask.

Thanks.

**Comment by MelleD** [comment]
Hey,

do yo have an example?

**Comment by Mattat01** [comment]
Not really, but I can show the current behaviour and try to explain.

The below image shows the Create Event rolled up (above the red line) and expanded (below). The calendar gets pushed down the page when the Create Event section is expanded, which is what it is supposed to do.

![image](https://github.com/user-attachments/assets/4eb249d0-8da8-47ad-9898-40588d1324b8)

Ideally, it would be great if the expanded card didn't push the calendar down, but became an overlay on top of the calendar. When the expanded section pushes elements around it messes up the layout of the page, but as an overlay it would leave all other cards in their proper place.

I fully expect that this is not possible, but just wanted to ask if it could be done. Assuming it cannot, I have changed my layout to a sidebar and I put the expanding section at the bottom of the sidebar so that there is noting below it to be pushed, which is ok.

Thanks.

**Comment by MelleD** [comment]
You can try the title card to integrate it

**Comment by Mattat01** [comment]
I'm not sure that putting it in another card will work. A better description of what I mean might be to say that it could act like a dropdown (like the AM/PM selector below).
![image](https://github.com/user-attachments/assets/2749d379-1823-461a-b0c5-fb7f4ba98b8e)

**Reply by MelleD** [reply, marked as answer]
Ah no drop-down is a little bit different and I think another component. Not sure if this is a good combination to do that in one component with the layout.

Only possibility is to try the title card and your days stay as title card and you slight down the create event. Maybe another possibility is the bubble card like an dialog.

**Comment by Mattat01** [comment]
Ok. Thanks for the pointers and thoughts. I'll take a bit more of a look at what I might be able to do.
